docs/dml-of-carbondata.md

```diff
@@ -350,17 +354,19 @@ CarbonData DML statements are documented here,which includes:
   This command allows you to load data using static partition.
 
   ```
-  LOAD DATA [LOCAL] INPATH 'folder_path' 
+  LOAD DATA INPATH 'folder_path'
```

Comment:
Local semantic is supported currently and not invalid. It should not be removed from the documentation.

Reply:
Yes, 'LOCAL' semantic is supported currently but it doesn't take effect, please see the code:
[CODE](https://github.com/apache/carbondata/blob/master/integration/spark2/src/main/scala/org/apache/spark/sql/parser/CarbonSpark2SqlParser.scala#L426)